docs/CONTRIBUTING.md

```diff
@@ -15,6 +15,27 @@ GitHub engineers tend to focus on areas of the project that are useful to GitHub
 
 Contributions to this project are [released](https://help.github.com/articles/github-terms-of-service/#6-contributions-under-repository-license) to the public under the [project's open source license](https://github.com/github/view_component/blob/main/LICENSE.txt).
 
+## Bugs and issues
+
+When opening an issue to describe a bug you're facing, it's helpful to replicate the bug. You can do this either by contributing failing tests in a pull request, or by creating a new repository that demonstrates the issue. You can do this as follows:
+
+1. Run `rails new --minimal view_component-bug-replica` in the console.
+2. Run `bundle add view_component` in the console. You may need to update your Gemfile and run `bundle install` to run against the version of `view_component` with which you're experiencing the issue.
```

Comment:
I don't know the right place to document it, but our style is to avoid 'you' as much as possible.

```suggestion
2. Run `bundle add view_component` in the console. If testing against a previous version of ViewComponent, update the `Gemfile` to that version and run `bundle install`.
```